// Last updated: 7/9/2026, 9:14:42 AM
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int j=0;
       for(int i =0;i<nums.length;i++){
            if(nums[i]%2==0){
                int temp =nums[i];
                nums[i]=nums[j];
                nums[j]=temp;
                j++;
            }
        }
    return nums;}
}